### PR TITLE
Feature/smhe 732 last gasp udp alarm set push udp alarm

### PR DESCRIPTION
**Pull request by harrymiddelburgetall**

### CODE REVIEW

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringconfiguration/SetPushSetupLastGasp.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+import org.opensmartgridplatform.cucumber.platform.smartmetering.support.ws.smartmetering.configuration.SetPushSetupLastGaspRequestFactory;
+import org.opensmartgridplatform.cucumber.platform.smartmetering.support.ws.smartmetering.configuration.SmartMeteringConfigurationClient;
+import org.opensmartgridplatform.shared.exceptionhandling.WebServiceSecurityException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class SetPushSetupLastGasp {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(SetPushSetupLastGasp.class);
```
Comment (kroesctrl) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringconfiguration/SetPushSetupLastGasp.java]: SLf4j

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringconfiguration/SetPushSetupLastGasp.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+import org.opensmartgridplatform.cucumber.platform.smartmetering.support.ws.smartmetering.adhoc.SmartMeteringAdHocRequestClient;
+import org.opensmartgridplatform.cucumber.platform.smartmetering.support.ws.smartmetering.adhoc.SmartMeteringAdHocResponseClient;
+import org.opensmartgridplatform.cucumber.platform.smartmetering.support.ws.smartmetering.configuration.SetPushSetupLastGaspRequestFactory;
+import org.opensmartgridplatform.cucumber.platform.smartmetering.support.ws.smartmetering.configuration.SmartMeteringConfigurationClient;
+import org.opensmartgridplatform.shared.exceptionhandling.WebServiceSecurityException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class SetPushSetupLastGasp {
```
Comment (kroesctrl) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringconfiguration/SetPushSetupLastGasp.java]: Convention Classname should end with ....Steps

File: osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/PushSetupObject.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Smart Society Services B.V.
+ * Copyright 2022 Alliander N.V.
```
Comment (kroesctrl) [inline osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/PushSetupObject.java]: Do not change copyright

**Review by kroesctrl — CHANGES_REQUESTED** [review]

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringconfiguration/SetPushSetupLastGasp.java (hunk 2 above)
Comment (harrymiddelburgetall) [reply integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringconfiguration/SetPushSetupLastGasp.java]: Okay, I was not aware of such convention. There are a lot of classes defining the steps not have their names ending with ...Steps but nevertheless renamed.

File: osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/PushSetupObject.java (hunk 3 above)
Comment (harrymiddelburgetall) [reply osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/PushSetupObject.java]: reverted

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringconfiguration/SetPushSetupLastGasp.java (hunk 1 above)
Comment (harrymiddelburgetall) [reply integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringconfiguration/SetPushSetupLastGasp.java]: replaced

**Comment by harrymiddelburgetall** [comment]
✔ Nightly: https://jenkins.fdp.osgp.cloud/view/Nightly%20Build/job/gxf-nightly-build/544/

**Review by kroesctrl — APPROVED** [review]

**Pull request merged**